js/os.ts

```diff
@@ -186,44 +186,85 @@ export function env(): { [index:string]: string } {
   return createEnv(res);
 }
 
+/** A class representing information about a given Path.  */
```

Comment:
/** A FileInfo describes a file and is returned by stat, lstat, statSync, lstatSync. */

(It's usually best to steal documentation phrasing from Golang - they have very lucid explanations.)